CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#[[
+This source file is part of the Swift System open source project
+
+Copyright (c) 2019 Apple Inc. and the Swift Numerics project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
```

Comment:
Change comment to System's header comment.

Reply:
Sure, that should be easy to do.